### PR TITLE
16.0 mig mis builder total committed purchase

### PR DESCRIPTION
**Pull request by dzungtran89**
Needs:
- https://github.com/OCA/mis-builder/pull/472

### CODE REVIEW

File: mis_builder_total_committed_purchase/__manifest__.py [hunk 1]
```diff
@@ -6,14 +6,14 @@
...
     "license": "AGPL-3",
     "author": "ACSONE SA/NV, Camptocamp, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/mis-builder-contrib",
     "depends": ["mis_builder", "purchase"],
     "data": [
         "data/mis_total_committed_purchase.sql",
-        "data/mis_total_committed_purchase_tag_rel.sql",
+        "data/mis_total_committed_purchase_analytic_account_rel.sql",
         "security/mis_total_committed_purchase.xml",
         "views/mis_total_committed_purchase.xml",
```
Comment (sonhd91) [inline mis_builder_total_committed_purchase/__manifest__.py:18]: Do we need access this view on UI ?

**Comment by sonhd91** [comment]
Minor review. not an expert but LGTM

File: mis_builder_total_committed_purchase/__manifest__.py (hunk 1 above)
Comment (dzungtran89) [reply mis_builder_total_committed_purchase/__manifest__.py:18]: yes, in order to view the source data

Comment (sonhd91) [reply mis_builder_total_committed_purchase/__manifest__.py:18]: I mean is there any way to access this view on UI ?

Comment (dzungtran89) [reply mis_builder_total_committed_purchase/__manifest__.py:18]: you can install the module `mis_builder_demo`, then changing the source of one label, then you can drilldown the data which is pointing to proper data source

image

**Pull request closed without merging**